style.css

```diff
@@ -509,6 +509,10 @@ button:hover {
   /* background-color: #1e1e1e; */
 }
 
+#change-tect-color {

```

Comment:
change id name. typo error found (change-text-color instead of change-tect-color)